.gitignore

```diff
@@ -6,6 +6,10 @@ stats.html
 screenshots
 .vscode
 
+packages/gestalt-core/dist/*
```

Comment:
@AlbertCarreras as a follow up, let's see if we can remove it.

Reply:
fixed